packages/app/src/app/pages/Dashboard/Content/routes/Recent/index.tsx

```diff
@@ -1,77 +1,88 @@
-import React from 'react';
-import { Helmet } from 'react-helmet';
-import { SelectionProvider } from 'app/pages/Dashboard/Components/Selection';
+import React, { useEffect } from 'react';
 import { useAppState, useActions } from 'app/overmind';
 import { sandboxesTypes } from 'app/overmind/namespaces/dashboard/types';
 import { Header } from 'app/pages/Dashboard/Components/Header';
 import { VariableGrid } from 'app/pages/Dashboard/Components/VariableGrid';
+import { SelectionProvider } from 'app/pages/Dashboard/Components/Selection';
+import { Helmet } from 'react-helmet';
 import {
   DashboardGridItem,
-  DashboardHeader,
-  DashboardSandbox,
+  DashboardTemplate,
   PageTypes,
 } from 'app/pages/Dashboard/types';
-import { getPossibleTemplates } from '../../utils';
 
 export const Recent = () => {
   const {
     activeTeam,
-    dashboard: { sandboxes, recentSandboxesByTime, getFilteredSandboxes },
+    dashboard: { viewMode, sandboxes },
   } = useAppState();
   const {
     dashboard: { getPage },
   } = useActions();
 
-  React.useEffect(() => {
+  useEffect(() => {
     getPage(sandboxesTypes.RECENT);
```

Comment:
Does `getPage` change when `activeTeam` changes by any chance and can we use that in the dependency array instead?

Reply:
Don't know exactly how the changes happen in overmind, as getPage is coming from the app state/dashboard. But I'd leave this as it is for now, it makes sense to have the fetching depend on the activeTeam.